### PR TITLE
feat(cli): forge bulk upload ecs state script

### PR DESCRIPTION
**Pull request by Kooshaba**
- add `mud bulkupload` cli script

### CODE REVIEW

File: packages/std-client/src/utils.ts [hunk 1]
```diff
@@ -56,9 +56,9 @@ export function getGameConfig(
   gameConfigComponent: Component<{ startTime: Type.String; turnLength: Type.String; actionCooldownLength: Type.String }>
 ) {
   const godEntityIndex = world.entityToIndex.get(GodID);
-  if (!godEntityIndex) return null;
+  if (!godEntityIndex) return;
```
Comment (alvrs) [inline packages/std-client/src/utils.ts:59]:
```suggestion
  if (godEntityIndex == null) return;
```
otherwise this fails if god happens to be at index 0

File: packages/network/src/workers/constants.ts [hunk 2]
```diff
@@ -6,4 +6,4 @@ export enum SyncState {
   LIVE,
 }
 
-export const GodID = "0x060D" as EntityID;
+export const GodID = "0x060d" as EntityID;
```
Comment (alvrs) [inline packages/network/src/workers/constants.ts:9]: why this change?

**Review by alvrs — APPROVED** [review]
lgtm

File: packages/network/src/workers/constants.ts (hunk 2 above)
Comment (Kooshaba) [reply packages/network/src/workers/constants.ts:9]: the ID comes back from on-chain lowercase

**Review by alvrs — APPROVED** [review]
